packages/components/src/components/ResourceTable/ResourceTable.js

```diff
@@ -38,7 +38,10 @@ const ResourceTable = props => {
         getRowProps,
         getTableProps
       }) => (
-        <TableContainer title={title} className="tkn--table">
+        <TableContainer
+          title={title}
+          className="tkn--table tkn--resource--table"
```

Comment:
```suggestion
          className="tkn--table tkn--resource-table"
```

`--` is used to separate variants or to indicate a child element, a single `-` is used otherwise